cmd/resetHeaderCheckCount.go

```diff
@@ -0,0 +1,55 @@
+// VulcanizeDB
+// Copyright © 2020 elizabethengelman
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
+	"github.com/makerdao/vulcanizedb/utils"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var blockNumber int
+
+// resetHeaderCheckCountCmd represents the resetHeaderCheckCount command
+var resetHeaderCheckCountCmd = &cobra.Command{
+	Use:   "resetHeaderCheckCount",
+	Short: "Resets header check_count for the given block number",
+	Long: `Resets check_count to zero for the given header so that the execute command may rechecked that header's logs
+in case one was missed.
+
+Use: ./vulcanizedb resetHeaderCheckCount --header-block-number=<block number>
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		SubCommand = cmd.CalledAs()
+		LogWithCommand = *logrus.WithField("SubCommand", SubCommand)
+		LogWithCommand.Infof("Updating check_count for header %v set to 0.", blockNumber)
+		return resetHeaderCount(int64(blockNumber))
```

Comment:
One of the downsides to using RunE over Run is that I haven't figured out how to include a SubCommand. I am thinking of just wrapping this error with SubCommand information for now.